js/main.js

```diff
@@ -31,23 +31,44 @@ getAPI(
   'https://api.pandascore.co/lol/champions?token=_g8je6_IZ8WrEPjlCgB-PG2588hD3Ume9yISokmfKl3uP-h5gGc&page[size]=100&page[number]=2'
 );
 
-function renderChamps() {
+function championDOMTree(i) {
```

Comment:
Instead of making these functions take in an argument of the index, can you make it take in an argument of the object for this champion? If it always has to pull its data from this specific array, this function isn't actually very reusable